index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+    <section class="head">
+      <div class="background-image-1"></div>
+      <ul class="status-bar">
+        <li class="time-text">1:00</li>
+        <li class="group-1">
+          <img class="inside" src="icons/Cellular-Connection.svg" alt="cellular" id="connection">
+          <img class="inside" src="icons/Wifi.svg" alt="Wifi" id="wifi">
+          <img class="inside" src="icons/Battery.svg" alt="battery" id="battery">
+        </li>
+      </ul>
+      <nav class="logo-placeholder">
+      <a href="#" class="logo">Maroun</a>
+      <a href="#"><img src="icons/menu.svg" alt="menu" id="menu"></a>
+      </nav>
+      <div class="headline">
+          <ul>
+        <li class="primary-text">I’m Maroun Glad to see you!</li>
+        <li class="primary-description">I’m a software developer! I can help you build a product , feature or website
+          Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</li>
```

Comment:
This is a paragraph and the proper HTML tag for paragraphs are `p` tags i suggest you wrap this section in a `p` tag for more information on the importance of using proper HTML semantic tags you can read this [Link](https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0)

Reply:
:heavy_check_mark: Fixed :blush: